google/resource-snippets/compute-v1/alternatives-firewall/tf/firewall.tf

```diff
@@ -0,0 +1,18 @@
+variable "deployment" {}
+variable "project_id" {}
```

Comment:
Please add the type and description to the two variables, see example below:
https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/google/resource-snippets/pubsub-v1/alternatives/tf/pubsub.tf#L7-L17

Reply:
Done.